config.js

```diff
@@ -0,0 +1,15 @@
+import { initializeApp } from 'firebase/app';
+import { getAuth, GoogleAuthProvider } from 'firebase/auth';
+const firebaseConfig = {
+  apiKey: 'AIzaSyBOTI15w1U5rJqX_cBZ8J-qa1Wb475klAo',
```

Comment:
apikey is a sensitive information so we want to hide it. I already made this config in utils/firebase.js and you need to make .env.local